src/lib/hooks/use-trade.ts

```diff
@@ -91,8 +91,8 @@ export const useTrade = () => {
         logTransaction(chainId ?? -1, hash, formatQuoteAnalytics(quote))
         setIsTransacting(false)
       } catch (error) {
-        console.log('Override?', override)
-        console.log('Error sending transaction', error)
+        console.info('Override?', override)
```

Comment:
```suggestion
```
removing since there's not much we can do from it

Reply:
I'd like to keep this one - as it's good to know if the user has been overriding the failed simulation.